packaging/windows/host/host.wxs

```diff
@@ -35,9 +34,13 @@
         </Directory>
     </Fragment>
     <Fragment>
-        <ComponentGroup Id="InstallFiles">
+        <ComponentGroup Id="InstallSharedHostandDetectionKeys">
```

Comment:
Thanks - I am not sure if the ordering issue can be fixed without creating a new hierarchy.